src/Sylius/Abstraction/StateMachine/src/CompositeStateMachine.php

```diff
@@ -39,35 +40,31 @@ public function __construct(
         $this->stateMachineAdapters = $stateMachineAdapters instanceof Traversable ? iterator_to_array($stateMachineAdapters) : $stateMachineAdapters;
     }
 
-    /**
-     * @throws \Exception
-     */
+    /** @throws StateMachineExecutionException */
```

Comment:
I believe it could be included in the interface rather than the implementation

Reply:
I think it is also good to have it in the implementation.